sherpa-onnx/csrc/online-model-config.h

```diff
@@ -38,14 +39,15 @@ struct OnlineModelConfig {
                     const OnlineParaformerModelConfig &paraformer,
                     const OnlineWenetCtcModelConfig &wenet_ctc,
                     const OnlineZipformer2CtcModelConfig &zipformer2_ctc,
-                    const std::string &tokens, int32_t num_threads, bool debug,
+                    const std::string &tokens, int32_t num_threads, int32_t warm_up, bool debug,
```

Comment:
Please also change the python counterpart in sherpa-onnx/python/csrc/online-model-config.cc.

Reply:
yes,I have added the same in recent commit, 
please let me know if it is sufficient.